charts/incubator/etesync/values.yaml

```diff
@@ -0,0 +1,104 @@
+image:
+  repository: victorrds/etesync
+  pullPolicy: IfNotPresent
+  tag: 0.9.1@sha256:a880ed256dd78f14fb8dc9ab2e72ca2acebd3b8a87db65ea038f30abed27d896
+
+podSecurityContext:
+  runAsUser: 373
+  runAsGroup: 373
+  fsGroup: 373
+
+etesync:
+  ALLOWED_HOSTS: "*"
+  AUTO_SIGNUP: false
+  LANGUAGE_CODE: "en-us"
+  SUPER_USER: "admin"
+  SUPER_PASS: "TestPassword"
+  SUPER_EMAIL: "admin@example.com"
+  DEBUG: true
+  SHELL_DEBUG: true
+  DEBUG_DJANGO: true
```

Comment:
I don't see the point of all this section, all these are already preset on `env`.
Am I missing something?

You can also use a section `secretEnv` for the `SUPER_*` envs
Which places them in a secret without the hassle

Reply:
> Am I missing something?

No, you're right. I'm still learning, haha.